CHANGELOG.md

```diff
@@ -4,12 +4,9 @@
 
 [Full Changelog](https://github.com/jwt/ruby-jwt/compare/v2.9.0...main)
 
-**Features:**
```

Comment:
This part should not be removed. Probably some merge issues when syncing with main...

Reply:
The changelog should only have additions in PRs. So would be great if you could bring back the red parts.

Reply:
Well I think I borked it up real good trying to reset from main and re-add the new version, I'll try again.

Reply:
:) good luck

Reply:
You don't need to worry about version numbers or anything. Just adding your entry above the `- Your contribution here`

Reply:
I think it's good now.

Reply:
Now the `- Your contribution here` was replaced. If you bring back that one we are good to go

Reply:
Added it back.

Reply:
Was just about to merge but there are some conflicts vs what is on main currently.

There was this 2.9.1 release in between that bumped the version in the changelog, think your branch is still based on the previous revision of `main`. If you would be so kind to do a rebase (and maybe also squashing the commits while at it)

Sorry for the back and forth and thank you for your patience.